model/driver/src/icon4py/model/driver/icon4py_driver.py

```diff
@@ -143,16 +151,20 @@ def time_integration(
         log.info(
             f"starting real time loop for dtime={self.dtime_in_seconds} n_timesteps={self._n_time_steps}"
         )
-        timer = Timer(self._full_name(self._integrate_one_time_step))
+        rest_timer = Timer(self._full_name(self._integrate_one_time_step), dp=6)
+        first_timer = Timer("first_time_step", dp=6)
         for time_step in range(self._n_time_steps):
+            timer = first_timer if time_step == 0 else rest_timer
+
             log.info(f"simulation date : {self._simulation_date} run timestep : {time_step}")
-            log.debug(
-                f" MAX VN: {np.abs(prognostic_states.current.vn.asnumpy()).max():.15e} , MAX W: {np.abs(prognostic_states.current.w.asnumpy()).max():.15e}"
-            )
-            log.debug(
-                f" MAX RHO: {np.abs(prognostic_states.current.rho.asnumpy()).max():.15e} , MAX THETA_V: {np.abs(prognostic_states.current.theta_v.asnumpy()).max():.15e}"
-            )
-            # TODO (Chia Rui): check with Anurag about printing of max and min of variables. Currently, these max values are only output at debug level. There should be namelist parameters to control which variable max should be output.
+            if __debug__:
+                log.debug(
```

Comment:
important to have this change for performance measurement in the greenline

Reply:
if you want to compare to fortran openacc or granule:
We noticed that there is an expensive debug print in the blueline with loglevel > 7?
So for performance run with Fortran ICON log level 1